src/Symfony/Component/DependencyInjection/Container.php

```diff
@@ -359,14 +358,19 @@ public function initialized($id)
     public function getServiceIds()
     {
         $ids = array();
+        $reversedMethodMap = array_change_key_case(array_flip($this->methodMap), \CASE_LOWER);
```

Comment:
I don't think the changes on this method are legitimate: `get*Service` are the only "source", methodMap must be in sync with the correspond list and we don't need to merge it.